torch/ao/quantization/fx/_model_report/model_report_visualizer.py

```diff
@@ -568,10 +567,14 @@ def generate_plot_visualization(self, feature_filter: str, module_fqn_filter: st
         if data_per_channel:
             ax.set_xlabel("First idx of module")
             # set the legend as well
-            # plot a seperate line for each channel
-            for index, channel_info in enumerate(y_data):
-                ax.plot(x_data, channel_info, label="Channel {}".format(index))
+            # plot a single line that is average of the channel values
```

Comment:
how is average for all channels useful to user?

Reply:
It will give them a general trend and if we want to show just one line of information, i felt this would be the most useful of having an average value be used.

The issue with showing all the different channels is that when you have 500+ channels, you really can't see anything cuz you have 500 lines on the plot.

Maybe for some types of information, this doesn't help as much, but for things like per_channel_weight_max or stats like that, getting an average max val across channels for each module and seeing that progression through the model can definately be useful in my opinion

Reply:
oh I mean when do they need to see this information, maybe can you document this any where? (as a non related example, when a user is trying to understand why we choose static quant v.s. dynamic quant, they might want to see some metric A)

Reply:
or in your answer, "getting an average max val across channels for each module and seeing that progression through the model can definately be useful in my opinion" can you talk in more detail about how exactly that is useful?

Reply:
well in the answer i provided, suppose users are trying to figure out why a set of input activations was classified as a non-stationary distribution or where the stationarity changes in a model. They can plot the average max and average min vals across the modules and that would give them a better sense of what is happening because the size of the batch ranges and epoch ranges per channel are what determine stationarity. 

Looking at the plots, they may realize that for a specific module C, the average input activation value max across channel sharply decreased across channels and the min sharply increased, leading to a lower stationarity (because batch range decrease -> stationarity decrease).

Reply:
Updated get_plot_visualization() method description to highlight reasoning a bit more.